app/views/articles/index.html.erb

```diff
@@ -1,2 +1,47 @@
-<h1>Articles#index</h1>
-<p>Find me in app/views/articles/index.html.erb</p>
+<div class="container">
+
+  <div class="row">
+    <div class="col-xs-12">
+      <h2>All articles</h2>
+    </div>
+  </div>
+
+  <div class="row">
+    <% @articles.each do |article| %>
+
+    <!-- One card per booking linking to the article page -->
+
+    <div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
+    <%= link_to article_path(article) do %>
+
+      <div class="card">
+
+        <h3><%= article.title %></h3>
+        <h4>By <%= article.user.name %> - created on <%= article.created_at.to_date %></h4>
+
+        <!-- Average review -->
+        <p>
+          <% average_rating = article.reviews.reduce(0){|sum, r| sum + r.rating} / article.reviews.length.to_f %>
+          <% 1.upto(5) do |i| %>
+            <% if i <= average_rating %>
+              <i class="fa fa-star" aria-hidden="true"></i>
+            <% elsif (i-1..i).include? average_rating  %>
+              <i class="fa fa-star-half-o" aria-hidden="true"></i>
+            <% else %>
+              <i class="fa fa-star-o" aria-hidden="true"></i>
+           <% end %>
+          <% end %>
+          <p><em>(<%= article.reviews.length %> reviews)</em></p>
+        </p>
+
+        <p><em><%= article.summary %></em></p>
+
+      <% end %>
+
+    <% end %>
+    </div>
```

Comment:
Yes! We need to draw a card together and agree which elements to place and where so that we can reshuffle in html if needed and agree on css